test/mocks.cpp

```diff
@@ -57,8 +58,9 @@ pnal_eth_handle_t * mock_pnal_eth_init (
 {
    pnal_eth_handle_t * handle;
 
-   handle = (pnal_eth_handle_t *)calloc (1, sizeof (pnal_eth_handle_t));
+   handle = (pnal_eth_handle_t *)calloc (1, 10 /*sizeof (pnal_eth_handle_t)*/);
```

Comment:
Was this intended?

Reply:
I agree this looks strange. See updated mock implementation.